dev/com.ibm.ws.concurrent.persistent_fat_configupdate/fat/src/com/ibm/ws/concurrent/persistent/fat/configupd/FATSuite.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.concurrent.persistent.fat.configupd;
+
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import com.ibm.websphere.simplicity.Machine;
+
+import componenttest.topology.impl.LibertyFileManager;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+
+@RunWith(Suite.class)
+@SuiteClasses({ PersistentExecutorConfigUpdateTest.class })
```

Comment:
how long does this config update test take to run? If it takes >5m, we should move some/all of the tests to FULL mode. (If all tests are moved to FULL, remember that a dummy AlwaysPassesTest needs to be added)